static/js/vue-cdr-access/src/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
+    <browse-display :browse-path="record_id"
+                    :records-per-page="20"></browse-display>
+  </div>
+</template>
+
+<script>
+import browseDisplay from './components/browseDisplay.vue'
+
+export default {
+  name: 'app',
+
+  components: {
+    browseDisplay
+  },
+
+  data() {
+    return {
+      record_id: `/listJson/${location.pathname.split('/')[2]}`
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+#app {
+  font-family: 'Avenir', Helvetica, Arial, sans-serif;
```

Comment:
Did we decide to use a different font family just for the browse?

Reply:
No, I'll get rid of it; a generated style